app/views/apis/show.html.erb

```diff
@@ -1,28 +1,32 @@
 <h2><%= @api.name %></h2>
 
 <p>
-  <%= @api.name %>
-  <%= @api.url %>
-  <%= @api.description %>
-  <br />
-  <% tags = "" %>
-    <% @api.tags.each do |tag| %>
-    <% tags += "#{tag.name} " %>
-    <% end %>
-  <%= "TAGS: #{tags} " %>
-</p>
+  <%= @api.url %> <br /><br />
```

Comment:
Gotta lose the `<br>`s here and get some divs up in this piece. Check out the Foundation docs on using the [grid](http://foundation.zurb.com/sites/docs/v/5.5.3/components/grid.html).